### DISCUSSION TITLE
/zh-cn/blog/posts/night_talk_001

### DISCUSSION THREAD

**Discussion by giscus** [Reverier-Xu/discussions#137] [Announcements]
# 音乐与时间与蓝色星球#01 夜话 - Reverier's Blog

note1: 随便写写。 note2: 标题 neta 自 ギターと孤独と蒼い惑星。 note3: 啊这篇不是写音乐的，👴只会点管乐，也对音乐没什么深入理解。 夜话 最早看到“夜话”这个词是在读了

https://blog.woooo.tech/posts/night_talk_001/

**Comment by spn21** [comment]
在线催更ing（x

**Reply by Reverier-Xu** [reply]
催我不如催☁️🈸🌞📓，我这emo log有啥看头（

**Reply by spn21** [reply]
😭😭😭
